src/main/java/appeng/integration/modules/NEIHelpers/NEISearchField.java

```diff
@@ -38,6 +38,16 @@ public boolean existsSearchField() {
         return getSearchField() != null;
     }
 
+    public String getEscapedSearchText(String text) {
+        final SearchField searchField = getSearchField();
+
+        if (searchField != null) {
```

Comment:
I'm not verry good at NEI's things, but can we change this to
``` java
        if(getSearchField() != null) {
        	
        }
```

cause this `searchField` seems never use?

Reply:
im pretty sure the jvm will do that already when there is only 1 instance of a variable being used

Reply:
done